packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -37,7 +38,10 @@ const gpuColorPalette = [
 class FrameFlameChart extends CoreElement {
   FrameFlameChart() : super('div') {
     flex();
+    layoutVertical();
     clazz('flame-chart');
+    element.style.backgroundColor = colorToCss(const Color(0xFFF3F3F3));
```

Comment:
any reason to not specify this style in the existing css for
flame-chart?
Might as well keep all the styles for an element in one place.

Reply:
I was just trying to do styling in dart for easier porting to hummingbird, but I can move it to css to keep it together.

Reply:
Actually I will just move the other styles from css to dart.

Reply:
fine to leave as is or move all the styles to dart.